src/code/views/node-edit-view.coffee

```diff
@@ -10,21 +10,53 @@ module.exports = React.createClass
   changeImage: (e) ->
     @props.onNodeChanged? @props.node, @props.node.title, e.target.value
 
+  addRemote: (e) ->
```

Comment:
Not sure, but this also seems like a concern that could get pushed out of the view.